### DISCUSSION TITLE
Selecting which columns are shown in a `DataGrid`

### DISCUSSION THREAD

**Discussion by ero-qt** [AvaloniaUI/Avalonia#16232] [Q&A]
Hi, is it possible to allow users to select which headers are shown in a `DataGrid`? Implementations of tables like that often have the ability for the user to right-click the header row and select which columns are shown. Will I have to create my own control from scratch for this?

**Comment by timunie** [comment]
No, you can just bind to DataGridColum.IsVisibke property

**Reply by ero-qt** [reply]
Bind what to it? I guess I want a context menu of some sort?

**Reply by maxkatz6** [reply]
You can, up to you. Such functionality can be built on top of DataGrid.
